spec/javascripts/components/projects/side_bar/side_bar_button_info.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SideBarButtonInfo from 'components/projects/SideBar/SideBarButtonInfo';
+
+describe('<SideBarButtonInfo />', () => {
+  it('renders the component', () => {
+    const wrapper = shallow(<SideBarButtonInfo description='' />);
+
+    expect(wrapper).toExist();
+  });
+
+  const descriptions = ['foo','bar','lorem ipsum'];
+
+  descriptions.forEach(description => {
```

Comment:
I don't think we need three hypothetical scenarios that behave exactly the same way, just one would be enough :smile: Also, it would be good that the test description describes _the feature_, not the result.